### PR TITLE
Auto rotation enabled

### PR DESCRIPTION
**Pull request by kwagyeman**

### CODE REVIEW

File: src/omv/main.c [hunk 1]
```diff
@@ -495,6 +496,9 @@ int main(void)
     // mounting the file-system to log errors (if any).
     if (first_soft_reset) {
         sensor_init_ret = sensor_init();
+        #if MICROPY_PY_IMU
+        if ((!sensor_init_ret) && (sensor_get_id() == OV7690_ID)) py_imu_init();
```
Comment (iabdalkader) [inline src/omv/main.c]: Why check for OV7690 ? It needs to be initialized if it's enabled, agree ?

File: src/omv/sensor.c [hunk 2]
```diff
@@ -433,7 +433,11 @@ int sensor_reset()
     sensor.hmirror       = false;
     sensor.vflip         = false;
     sensor.transpose     = false;
+    #if MICROPY_PY_IMU
+    sensor.auto_rotation = sensor.chip_id == OV7690_ID;
```
Comment (iabdalkader) [inline src/omv/sensor.c]: Maybe it's better to move this to ov7690_init ?

**Comment by kwagyeman** [comment]
You need to create a new target for the new camera then.

File: src/omv/sensor.c (hunk 2 above)
Comment (kwagyeman) [reply src/omv/sensor.c]: It can't be in init because reset would clear it...

Comment (kwagyeman) [reply src/omv/sensor.c]: Oh, I see... like the color_palette.

Comment (kwagyeman) [reply src/omv/sensor.c]: No, I think it makes sense in reset...

Comment (iabdalkader) [reply src/omv/sensor.c]: I meant here:
https://github.com/openmv/openmv/blob/master/src/omv/ov7690.c#L616

Comment (kwagyeman) [reply src/omv/sensor.c]: It doesn't really make sense to set it there. Instead can you merge the PR and then create a micro target and then we it being set correctly into the micro target.

**Pull request merged**